examples/apexlog/main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"errors"
+	"time"
+
+	"github.com/airbrake/gobrake/v5"
+	"github.com/airbrake/gobrake/v5/apexlog"
+	"github.com/apex/log"
+)
+
+func main() {
+	// Initialize the airbrake notifier
+	airbrake := gobrake.NewNotifierWithOptions(&gobrake.NotifierOptions{
+		ProjectId:   105138,                             // use the ProjectId of you project
+		ProjectKey:  "fd04e13d806a90f96614ad8e529b2822", // use the ProjectKey of you project
+		Environment: "production",
+	})
+
+	// Close the airbrake notifier
+	defer airbrake.Close()
+
+	// Create the airbrake handler for "github.com/apex/log"
+	// Insert the airbrake notifier instance and set the severity level.
+	// The acceptable severity levels are "debug" "info", "warn", "error", and "fatal".
+	// If the severity level is not recognized, "error" severity is used.
+	airbrakeHandler := apexlog.New(airbrake, "error")
+
+	log.SetLevel(log.DebugLevel)
+
+	// Set the airbrake handler in the log
+	log.SetHandler(airbrakeHandler)
+
+	ctx := log.WithFields(log.Fields{
+		"file": "something.png",
+		"type": "image/png",
+		"user": "tobi",
+	})
+
+	for range time.Tick(time.Millisecond * 500) {
+		ctx.Info("upload")
+		ctx.Info("upload complete")
+		ctx.Warn("upload retry")
+		ctx.WithError(errors.New("unauthorized")).Error("upload failed")
+		ctx.Errorf("failed to upload %s", "img.png")
+	}
+}
```

Comment:
It would be nice to print a message that encourages the user to check their project on Airbrake after the logs were sent.

Reply:
Please explain it briefly. Not sure what to write.

Reply:
```go
fmt.Println("Go check your Airbrake dashboard at https://airbrake.io/projects/123 to see these log messages")
```